### ISSUE TITLE
[Invoices] Specific format for associations

### ISSUE THREAD

**Issue by audez** [openfoodfoundation/wishlist#392]
## What is the need / problem?
One of our hubs has requested a specific format for the invoices. As an association (non profit organization), they can't do invoices, so it would be a "Purchase order” (“Bon de commande” in French) instead of “Invoice”.
This format could also be of interest for other associations. 

## Which type of users does this problem affect (and how many, if known)?
_- Shoppers_
_- Hub Managers (MCFEs)_
_- Producers_

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]
Changes to be done:
–	Display “Purchase order” instead of “Invoice” ( in French “Bon de commande” instead of “Facture”)
–	Display the shop address instead of the shop legal address
–	Don’t display the region/province
–	Don’t display ACN
–	Don’t display “billing address”
–	First name : capitalized first letter 
–	Name : All caps ; font: 48 + bold
–	Phone number : put spaces every two digits, font: 12 + bold
–	Remove the “Payment summary” an “Payment description” block
–	 before the payment description, add text “Payment: on the spot,”

## Connected wishlist and discovery discussions* [list precedent discussions]
https://openfoodnetwork.slack.com/archives/CG7NJ966B/p1670793072875289

### Before 
<img width="588" alt="Capture d’écran 2022-12-26 à 15 33 13" src="https://user-images.githubusercontent.com/28386648/209559200-bdf95896-e2a7-4bf3-95a0-2900d6e43710.png">

### After
<img width="664" alt="Capture d’écran 2022-12-26 à 15 29 40" src="https://user-images.githubusercontent.com/28386648/209559014-2e7cc2e1-31d2-4506-b0f8-7b165a450929.png">

**Comment by RachL** [comment]
@audez I don't think we can apply this to all non profit project : actually a non profit can issue invoices like any company. 

This sentence:

>As an association (non profit organization), they can't do invoices

is not legally true.

So we are looking here at introducing a different document, with no link to invoices.

Do you have more info on the underlining need? In which context are they using these documents? A "purchase order" is usually sent to producer so they can prepare orders.

**Comment by audez** [comment]
> So we are looking here at introducing a different document, with no link to invoices.

Yes it's another document

> Do you have more info on the underlining need? In which context are they using these documents? A "purchase order" is usually sent to producer so they can prepare orders.
They are using it for accounting and to provide to clients. I'm not sure 'purchase order' is the right terminology in English, but it's "bon de commande" in French. And a "bon de commande"/purchase order is a document between a seller and a buyer, with details of the order. It can be used with suppliers but also with clients. 

> is not legally true.
Right, an association can do invoices. The hub told me they don't do it/need it though. I'll edit it in the issue

**Comment by RachL** [comment]
> between a seller and a buyer

Exactly but they are also the buyer from their producer and in this case I wonder if the template they've chosen can really work for BtoB: like ACN is definitely needed, same as address from supplier and for the shop it must be the legal address of the buyer not the address of the shop, which can be a pick-up point.

Are you sure they are going to use it for producers as well? If it is just for shoppers, how do they communicate the document to them? Print or email? Do they use it for packing?

Do they need a PDF for accounting? Wouldn't a report gathering all orders be more handy for accounting?

**Comment by audez** [comment]
No they won't use it for producers, they need it for their clients. I was just responding to your comment:
> A "purchase order" is usually sent to producer so they can prepare orders.

They will use it as they would use an invoice, the format is fine for them, it's just the changes described; yes being able to print or download; 

> Wouldn't a report gathering all orders be more handy for accounting?

no apparently that's what they need.

**Comment by RachL** [comment]
> They will use it as they would use an invoice

And how do they use invoice exactly? I'm asking because some people use them as invoices, other as packing sheet, some as a description that is printed and placed in the basked the shopper is coming to pick up.

From what I understand so far  - so I could be wrong - but I wonder if this can be really done within OFN.

Order form / purchase order are on the todo list, but in priority for the relation between producer and hub. It's a bit linked to the work we did on delivery note a couple of years ago: https://community.openfoodnetwork.org/t/inception-session-delivery-note/1286

Also for a proper order form should not be changed when the order is edited. The order form is what the buyer ordered, the delivery note what they have received. 

So if OFN implements an order form, the spec will really differ from what they are looking for. We can estimate this work and do a parallel estimate on creating a N8N flow just for this user with the taylor-made document they want at the end and let them compare.

A shortcut would be to straight do the estimate with N8N (or equivalent solution) as it's going to be cheaper than the other option anyway. If they agree to fund it, it's an opportunity to iterate and learn from this document use.

@lin-d-hop what do you think?

**Comment by audez** [comment]
ok I understand better the different issues that it's rising. I will set up another meeting with them to try to clarify this!
